### PR TITLE
feat(hosting.ftp): put forward SFTP information

### PR DESCRIPTION
**Pull request by lucas-chaigne**
ref: DTRSD-43496

Signed-off-by: Lucas Chaigne <lucas.chaigne@ovhcloud.com>

| Question         | Answer
| ---------------- | ---
| Branch?          | `develop`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | #DTRSD-43496
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

## Description

- Add documentation link in ftp tab of webhosting manager so user can have information on how to connect to its hosting in ftp, sftp or ssh.
- In webhosting manager, ftp-ssh tab, for pro and perf offers : user table is updated to add sftp information
- In webhosting manager, FTP-SSH tab, user modification and creation are updated so that user better understands options related to SFTP.
- In webhosting manager, FTP link to the hosting is removed from useful link. The goal is to push customer to use SFTP instead of FTP (more secure protocol). SFTP link is not added to useful link since browser do not support it.
- In webhosting manager, FTP-SSH page, information panel for pro/perfo hostings :
    - connection information (server and ports) for FTP, SFTP and SSH connection is displayed
    - redundant information is removed (distinction ssh/ftp primary login, direct ssh link and cluster IP address)

## Related

- [x] #5989
- [x] #5991
- [x] #5992
- [x] #5993

### :flags: Translation

- [x] TRANS-40939

### CODE REVIEW

File: packages/manager/apps/web/client/app/hosting/ftp/FTP.html [hunk 1]
```diff
@@ -10,6 +10,19 @@
 
             <p data-translate="hosting_tab_FTP_dashboard_information"></p>
 
+            <div class="mb-4">
+                <span class="fa fa-life-ring mr-2" aria-hidden="true"></span>
+                <span data-translate="hosting_tab_FTP_guide_help"></span>
+                <a
+                    data-ng-href="{{:: ctrlHostingTabFTP.guide}}"
+                    target="_blank"
+                    title="{{hosting_guide_help}} {{ 'common_newtab' | translate }}"
```
Comment (JeremyDec) [inline packages/manager/apps/web/client/app/hosting/ftp/FTP.html:19]:
```suggestion
                    data-ng-href="{{:: ctrlHostingTabFTP.guide}}"
                    target="_blank"
                    title="{{hosting_guide_help}} {{ 'common_newtab' | translate }}"
                    rel="noopener"
```

**Review by JeremyDec — CHANGES_REQUESTED** [review]

File: packages/manager/apps/web/client/app/hosting/ftp/FTP.html (hunk 1 above)
Comment (lucas-chaigne) [reply packages/manager/apps/web/client/app/hosting/ftp/FTP.html:19]: Hello,

thanks for the review. What's the advantage to have both target="_blank" and rel="noopener" ?

Comment (JeremyDec) [reply packages/manager/apps/web/client/app/hosting/ftp/FTP.html:19]: Hello, it is for security reason. When you click on a link, which opens a new tab (as you did by setting `target="_blank")`, by default the opened tab keeps a reference to the source window. This adds vulnerability as the opened new tab may manipulate data corresponding to the original document through the object `Window.opener`.

For more information, take a look at https://developer.mozilla.org/en-US/docs/Web/HTML/Link_types/noopener

Comment (lucas-chaigne) [reply packages/manager/apps/web/client/app/hosting/ftp/FTP.html:19]: Thanks for the info, pull request is updated with the change requested

**Review by JeremyDec — APPROVED** [review]

**Review by antleblanc — DISMISSED** [review]

**Review by JeremyDec — APPROVED** [review]

**Review by antleblanc — APPROVED** [review]

**Comment by antleblanc** [comment]
Pulling translations.

**Comment by antleblanc** [comment]
Resolve existing conflicts.

```txt
packages/manager/apps/web/client/app/hosting/dashboard/translations/Messages_de_DE.json
packages/manager/apps/web/client/app/hosting/dashboard/translations/Messages_en_GB.json
packages/manager/apps/web/client/app/hosting/dashboard/translations/Messages_es_ES.json
packages/manager/apps/web/client/app/hosting/dashboard/translations/Messages_it_IT.json
packages/manager/apps/web/client/app/hosting/dashboard/translations/Messages_pl_PL.json
packages/manager/apps/web/client/app/hosting/dashboard/translations/Messages_pt_PT.json
```

**Review by antleblanc — APPROVED** [review]

**Pull request merged**